packages/components/Button/src/Button.tsx

```diff
@@ -5,11 +5,11 @@ import { ActivityIndicator } from '@fluentui-react-native/experimental-activity-
 import { buttonName, ButtonType, ButtonProps } from './Button.types';
 import { TextV1 as Text } from '@fluentui-react-native/text';
 import { stylingSettings, getDefaultSize } from './Button.styling';
-import { compose, mergeProps, withSlots, UseSlots, memoize } from '@fluentui-react-native/framework';
+import { compose, mergeProps, withSlots, UseSlots } from '@fluentui-react-native/framework';
 import { useButton } from './useButton';
 import { Icon } from '@fluentui-react-native/icon';
 import { createIconProps, IPressableState } from '@fluentui-react-native/interactive-hooks';
-import { extractOuterStylePropsAndroid } from './utils.android';
+import { extractOuterStylePropsAndroid } from './ExtractStyle.android';
```

Comment:
Ah so doing it this way (importing the .android file directly) means it will end up getting included in all platform bundles anyway. At this point I think it's worth just renaming the file